src/components/messenger/list/index.tsx

```diff
@@ -227,22 +205,6 @@ export class Container extends React.Component<Properties, State> {
             />
           </div>
         )}
-
-        <FeatureFlag featureFlag='enableRewards'>
-          <div {...cnMessageList('rewards-container', !this.props.includeUserSettings && 'center')}>
-            <RewardsContainer
-              meowPreviousDay={this.props.meowPreviousDay}
-              isRewardsLoading={this.props.isRewardsLoading}
-              isMessengerFullScreen={this.props.isMessengerFullScreen}
-              showRewardsInTooltip={this.props.showRewardsInTooltip}
-              showRewardsInPopup={this.props.showRewardsInPopup}
-              isFirstTimeLogin={this.props.isFirstTimeLogin}
-              onRewardsPopupClose={this.props.rewardsPopupClosed}
-              onRewardsTooltipClose={this.props.rewardsTooltipClosed}
-              hasLoadedConversation={this.props?.conversations[0]?.hasLoadedMessages}
-            />
```

Comment:
so we're not gonna implement rewards in the future? Or will it 'not' be a part of messenger list?

Reply:
Yeah, I thought best to remove the RewardsContainer from here as the Rewards will be a subcomponent of a new panel (profile panel). 

https://www.figma.com/file/btwltL4Os8hAy3eS3pxaa6/ZERO-Messenger-Web-%7C-Glass?type=design&node-id=12361-20661&mode=design&t=8qNSkEFnhIoF6nTR-4

Similar thinking to how we've added `<GroupManagement />`. Something like `<UserProfile />` would contain the rewards container....Have a look at <GroupManagement />` in messenger list, let me know if that makes sense